static/templates/settings/organization_profile_admin.hbs

```diff
@@ -71,7 +71,7 @@
         </h3>
         <div class="deactivate-realm-section">
             <div class="input-group">
-                <button class="button rounded btn-danger" id="deactivate_realm_button">
+                <button class="button rounded" id="deactivate_realm_button">
```

Comment:
@ligmitz was thinking you just need to do :
```suggestion
                <button class="button rounded {{#if is_owner}} class-making-this-button-non-gray {{/if}}" id="deactivate_realm_button">
```

We don't want to do this via JS unless absolutely necessary.

Also, I think you are confused about the user permissions, see https://zulip.com/help/deactivate-your-organization

Reply:
@amanagr Okay , I wasn't aware about all the features of hbs ! I will execute it this way.